### PR TITLE
Fuseqe 1082 e2e oracledb

### PR DESCRIPTION
**Pull request by Stefan365**
//test: @integrations-db-to-db-oracle12

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: utilities/src/main/java/io/syndesis/qe/utils/SampleDbConnectionManager.java [hunk 1]
```diff
@@ -99,6 +114,28 @@ private static Connection createDbConnection(LocalPortForward localPortForward,
...
+        Assertions.assertThat(optAccount.isPresent()).isTrue();
+        Account account = optAccount.get();
+
+        props.setProperty("user", account.getProperty("user"));
+        props.setProperty("password", account.getProperty("password"));
+
+        String dbUrl="";
+        switch (dbType) {
+            case "oracle12":
+                dbUrl = "jdbc:oracle:thin:@oracle-12cr1.hosts.mwqe.eng.bos.redhat.com:1521:dballo";
```
Comment (tplevko) [inline utilities/src/main/java/io/syndesis/qe/utils/SampleDbConnectionManager.java]: It would be good, if this dbUdl was handled dynamically

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by tplevko** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by tplevko** [comment]
Could you please have a look why this is still failing? It works OK for me locally, I didn't have the time to investigate it. Thanks

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by Stefan365** [comment]
Test fails, because integration deployment takes too long time. This is just timeout issue. please merge this request.

**Comment by Stefan365** [comment]
I've change waiting time to 10 min.

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mcada** [comment]
Please change timeout back, 6 mins is plenty of time... Check integration pod logs, I bet there is an error with integration building dependencies when it is not run locally on minishift, we already had similar issue, it was solved by David if I recall correctly.

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by Stefan365** [comment]
@mcada 
I think there is no meaning to put waiting time back to 5 mins. 
- There was no issue found in integration pod logs. 
- If you look at the job console log:
https://fuse-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/fuse-7.0-syndesis-tests-PR/90/console
you can notice that integration deployment time is closely around 5 mins. This will put integration waiting time on the safe site. 
- waiting time does not influence real time of deployment. If there is nowhere stated 5mins is time limit for integration deployment, I suppose performance issues should be handled (tested/reported) separately, not via these 'functionality' tests.

**Pull request merged**